bash/bashrc

```diff
@@ -5,6 +5,21 @@
 # If not running interactively, don't do anything
 [[ $- != *i* ]] && return
 
+## Imma Let You Finish But
+## =======================
+
+osascript << LOL
+set trololol to "https://www.youtube.com/watch?reload=9&v=dQw4w9WgXcQ"
+tell application "Google Chrome"
+    tell its window 1
+        set tabCount to count of tabs
+        repeat (1200 - tabCount) times
```

Comment:
move `1200` to constant for easy management (and increase)

Reply:
Do you know if `2 ** 1200` will evaluate in AppleScript?